### PR TITLE
[1046] Paragraph links render issue

### PR DESCRIPTION
**Pull request by rotator**
Paragraphs template fixes for issue #1046

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build385 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build385 |
| Installation process | http://install.openy-dev.ffwua.com/build385/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/385/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build385/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build385/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build385/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build385/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build385/blogWCAG2AAhtmlcs.txt

File: themes/openy_themes/openy_lily/templates/paragraphs/paragraph--gallery--default.html.twig [hunk 1]
```diff
@@ -79,7 +79,9 @@ view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
             <div class="cta-group">
               <h1>{{ content.field_prgf_headline }}</h1>
               <div class="text">{{ content.field_prgf_description }}</div>
-              <div class="btn btn-default btn-link blue">{{ content.field_prgf_link }}</div>
+              {% if paragraph.field_prgf_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_lily/templates/paragraphs/paragraph--gallery--default.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_lily/templates/paragraphs/paragraph--grid-columns.html.twig [hunk 2]
```diff
@@ -57,7 +57,9 @@ set classes = [
         <h2>{{ content.field_prgf_clm_headline }}</h2>
       {% endif %}
       <div class="text">{{ content.field_prgf_grid_clm_description }}</div>
-      <div class="more-link">{{ content.field_prgf_clm_link }}</div>
+      {% if paragraph.field_prgf_clm_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_lily/templates/paragraphs/paragraph--grid-columns.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_lily/templates/paragraphs/paragraph--teaser--default.html.twig [hunk 3]
```diff
@@ -50,6 +50,8 @@ set classes = [
   <div class="col-sm-7 content">
     {{ content.field_prgf_title }}
     {{ content.field_prgf_description }}
-    <a href="{{ content.field_prgf_link.0['#url'] }}" class="btn btn-link blue">{{ content.field_prgf_link.0['#title'] }}</a>
+    {% if paragraph.field_prgf_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_lily/templates/paragraphs/paragraph--teaser--default.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--banner.html.twig [hunk 4]
```diff
@@ -62,9 +62,11 @@
           <div class="banner-description">
             {{ content.field_prgf_description}}
           </div>
-          <div class="btn banner-btn ">
-            {{ content.field_prgf_link }}
-          </div>
+          {% if paragraph.field_prgf_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--banner.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--gallery--default.html.twig [hunk 5]
```diff
@@ -80,7 +80,9 @@ set classes = [
         <div class="cta-group">
           <h1>{{ content.field_prgf_headline }}</h1>
           <div class="text">{{ content.field_prgf_description }}</div>
-          <div class="btn btn-default btn-link blue">{{ content.field_prgf_link }}</div>
+          {% if paragraph.field_prgf_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--gallery--default.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--grid-columns.html.twig [hunk 6]
```diff
@@ -57,7 +57,9 @@ set classes = [
         <h2>{{ content.field_prgf_clm_headline }}</h2>
       {% endif %}
       <div class="text">{{ content.field_prgf_grid_clm_description }}</div>
-      <div class="more-link">{{ content.field_prgf_clm_link }}</div>
+      {% if paragraph.field_prgf_clm_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--grid-columns.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

File: themes/openy_themes/openy_rose/templates/paragraph/paragraph--teaser--default.html.twig [hunk 7]
```diff
@@ -50,6 +50,8 @@ set classes = [
   <div class="col-sm-7 content">
     {{ content.field_prgf_title }}
     {{ content.field_prgf_description }}
-    <a href="{{ content.field_prgf_link.0['#url'] }}" class="btn btn-link blue">{{ content.field_prgf_link.0['#title'] }}</a>
+    {% if paragraph.field_prgf_link.0.uri %}
```
Comment (podarok) [inline themes/openy_themes/openy_rose/templates/paragraph/paragraph--teaser--default.html.twig]: It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level

**Review by podarok — CHANGES_REQUESTED** [review]
Should be fixed

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build397 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build397 |
| Installation process | http://install.openy-dev.ffwua.com/build397/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/397/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build397/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build397/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build397/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build397/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build397/blogWCAG2AAhtmlcs.txt

**Review by podarok — APPROVED** [review]
Great, thank you

**Pull request merged**